ci/rpm_build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Script for buiding RPMs in a chroot
+# If the STAGE_NAME environment variable is present the script will
+# attempt to parse it to determine what distribution to build for.
+#
+# For manual testing, you can set the environment variables CHROOT_NAME
+# and TARGET can be set.
+#
+# Default is to build for CentOS 7.
+
+set -ex
+
+mydir="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+ci_envs="$mydir/ci/parse_ci_envs.sh"
+if [ -e ${ci_envs}"" ]; 
```

Comment:
(style)  trailing whitespace
(lint) Did you forget the 'then' for this 'if'? [SC1049]
(lint) Couldn't parse this if expression. [SC1073]